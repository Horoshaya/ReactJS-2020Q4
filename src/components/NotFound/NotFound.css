.wrapper {
  display: flex;
  align-items: center;
  justify-content: center;
  flex-direction: column;
  height: 100vh;
}

.title {
  color: #ffffff;
  font-size: 20px;
}

.button {
  padding: 10px 15px;
  color: #f65261;
  text-transform: uppercase;
  background-color: #232323;
  border: 1px solid #f65261;
  border-radius: 5px;
  cursor: pointer;
  transition-duration: 0.3s;
}

.button:hover {
  color: #ffffff;
  background-color: #f65261;
}
