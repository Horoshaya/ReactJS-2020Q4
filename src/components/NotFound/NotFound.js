import React from 'react';
import styles from './NotFound.css';
import { Link } from 'react-router-dom';

const NotFound = () => {
  return (
    <div className={styles.wrapper}>
      <h2 className={styles.title}>Page not found</h2>
      <Link to="/" className={styles.button}>
        go back to home
      </Link>
    </div>
  );
};

export default NotFound;
