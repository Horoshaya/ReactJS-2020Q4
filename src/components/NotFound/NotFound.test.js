import React from 'react';
import rerender from 'react-test-renderer';
import NotFound from './NotFound';
import { BrowserRouter as Router } from 'react-router-dom';

describe('when NotFound', () => {
  test('then snapshot created', () => {
    const component = rerender.create(
      <Router>
        <NotFound />
      </Router>,
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
