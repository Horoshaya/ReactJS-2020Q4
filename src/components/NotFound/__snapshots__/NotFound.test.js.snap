// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`when NotFound then snapshot created 1`] = `
<div>
  <h2>
    Page not found
  </h2>
  <a
    href="/"
    onClick={[Function]}
  >
    go back to home
  </a>
</div>
`;
